src/foam/core/reflow/Console.js

```diff
@@ -246,7 +246,7 @@ foam.CLASS({
       },
       code: function() {
         let confirmationModal = this.ConfirmationModal.create({
-          title: `Are you sure you want to reset the view ?`,
+          title: `Unsaved changes will be lost, are you sure you want a New Reflow page?`,
```

Comment:
Single quote ' is faster than ` if you aren't using `'s multi-line or interpolation features.